smart_html_server/main.py

```diff
@@ -1,11 +1,17 @@
+import sys
+import os
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```

Comment:
This is because the main we started is not in the root directory, which can cause ```__Init__``` Invalid, so I think we can do this for now?

Reply:
I find we are using npm to run the dev server locally, is it possible to set PYTHONPATH environment variable in node script instead of setting here. We may need to use WSGI server in prod instead of flask buildin server

Reply:
You are right, using environment variables is a better choice.

But even if ASGI/WSGI is used, I think this problem will still not be solved during development (perhaps just moving the problem to the ASGI/WSGI server, of course, if ASGI/WSGI server script is in the root directory, this problem will not occur)

Reply:
Understood. It seems necessary to modify the main function, as its current form is not suitable for a Flask application in a production environment. I'll substitute it with a dev_server.py instead.

Reply:
We can complete ```dev_server.py ``` later,, but the current environment variables means are definitely much better than my previous method.